addon/components/fast-action.js

```diff
@@ -1,13 +1,25 @@
 import Ember from 'ember';
 import layout from '../templates/components/fast-action';
 
-const { SafeString } = Ember.Handlebars;
+const {
+  String: {
```

Comment:
We should be using the new imports here

Reply:
no nothing was from ember init - i'm thinking we should close and redo - i just hacked from memory - everything - including chrome --no-sandbox with travis, and the safe string fn.. 

let me 30 min (that would have been enough to do it right) aside, and circle back around

thanks for the sanity check :)

Reply:
We should use `ember init` or `ember-cli-update` to get the real set of changes.